### PR TITLE
Add book list view

### PR DESCRIPTION
**Pull request by sebasdeldi**
image

- Adds a new view for the books lists
- Moves reusable mixins to the utils.scss file

### CODE REVIEW

File: scss/list.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  max-width: 900px;
+  margin: 50px auto;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+  .mini-card {
+    width: 100%;
+    max-width: 198px;
+    height: 300px;
```
Comment (jjuli93) [inline scss/list.scss:15]: I think it is not necessary to set the height here

File: scss/list.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    .author {
+      font-size: 14px;
+      color: $nevada-gray;
+    }
+
+    .mini-cover-image {
+      width: 100%;
+      max-width: 142px;
```
Comment (jjuli93) [inline scss/list.scss:33]: why this max-width?

File: list.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <meta charset="utf-8">
+    <title>List</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="cards-container">
+      <div class="mini-card">
+        <img class="mini-cover-image" src="img/book-cover.png">
```
Comment (jjuli93) [inline list.html:13]: make sure to always add an alt to images: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

File: scss/list.scss (hunk 2 above)
Comment (sebasdeldi) [reply scss/list.scss:33]: I miss understood something listed in the tech guides

**Pull request merged**